### PR TITLE
Remove dataquery stuff

### PR DESCRIPTION
**Pull request by ralphbean**
@ianweller and I have been talking about this on and off, but this should
remove all of the authn/dataquery stuff that has been around for a while, but
hasn't been working.

### CODE REVIEW

File: createdb.py [hunk 1]
```diff
@@ -1,7 +0,0 @@
-import os
-
-if not os.environ.get("DATAGREPPER_CONFIG"):
-    os.environ["DATAGREPPER_CONFIG"] = '../development.cfg'
-
-from datagrepper.app import db
-db.create_all()
```
Comment (pypingou) [inline createdb.py]: No more createdb ?

Comment (ralphbean) [inline createdb.py]: No more db.

Comment (pypingou) [inline createdb.py]: Nevermind no need indeed :)

**Comment by pypingou** [comment]
:+1:

**Comment by ianweller** [comment]
:+1: :boom: :ship:

**Pull request merged**